features/bootstrap/FeatureContext.php

```diff
@@ -155,6 +157,41 @@ public function itsOutputShouldContain(PyStringNode $string)
         PHPUnit_Framework_Assert::assertContains((string) $string, $this->getOutput());
     }
 
+    /**
+     * @Given the following logger settings:
+     */
+    public function theFollowingLoggerSettings(PyStringNode $string)
+    {
+        $lines = preg_split("/\\r\\n|\\r|\\n/", $string->getRaw());
```

Comment:
Tried that and the result was one line returned from explode when on Windows platform. I can continue to play around with other options.